src/Symfony/Component/DependencyInjection/Argument/RewindableGenerator.php

```diff
@@ -29,4 +35,13 @@ public function getIterator()
 
         return $g();
     }
+
+    public function count()
+    {
+        if (is_callable($count = $this->count)) {
```

Comment:
we need it as calling `$this->count()` would fail below, but could be declared inside the if indeed